src/test/resources/test.properties

```diff
@@ -0,0 +1,3 @@
+spring.data.mongodb.uri=\
+  mongodb+srv://root:toor@rocinante.nrixl.mongodb.net/test?retryWrites=true&w=majority
```

Comment:
For test purposes it's okay, but not for production. Please do not push credentials in your production code.

Reply:
I know, that's why it's root:toor. Need to set it up properly but need some time.